mysql-test/suite/mariabackup/deprecated_options.result

```diff
@@ -0,0 +1,485 @@
+# Testing mariabackup --help option
+# Executing mariabackup --help command
+Open source backup tool for InnoDB and XtraDB
+
+Copyright (C) 2009-2015 Percona LLC and/or its affiliates.
+Portions Copyright (C) 2000, 2011, MySQL AB & Innobase Oy. All Rights Reserved.
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation version 2
+of the License.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You can download full text of the license on http://www.gnu.org/licenses/gpl-2.0.txt
+
+Usage: C:\GitHub-local-machine-testing\server\bld\extra\mariabackup\RelWithDebInfo\mariadb-backup.exe [--defaults-file=#] [--backup | --prepare | --copy-back | --move-back] [OPTIONS]
```

Comment:
Appreciate the effort made to add this test case. As you see with this line of the test result, and the few lines below, the result is going to depend on the environment where the test case is run.

Also test cases on largely help output can quickly change on any documentation fixes.

For something as simple as a deprecating an option, no test case is needed.

So please remove the test cases.

Reply:
Thanks @grooverdan, I was discussing this issue on channel with Brandon, he said it's a client side change so to prove my changes I need to write test but not an issue I'll remove it.

Reply:
You've changed the help output. If it was more substantial change in behaviour then a test case would be required.